tests/testthat/test-formatters.R

```diff
@@ -537,3 +537,15 @@ mpf <- MatrixPrintForm(strings = strs, spans = spans, aligns = aligns,
 
 expect_equal(length(grep("spn_val", toString(mpf))),
              1L)
+
+testthat::test_that("fmt_config works as expected", {
+  x <- fmt_config()
+  expect_identical(slot(x, "format"), NULL)
```

Comment:
isnt the standard "xx" for format?

Reply:
I think the default is `NULL` since it has to generalize to both numeric and non-numeric formats. From what I can tell `format_value`, `analyze`, etc. all use `NULL` as default.